plugins/SitesManager/API.php

```diff
@@ -337,6 +338,10 @@ public function getSitesWithAdminAccess($fetchAliasUrls = false, $pattern = fals
             }
         }
 
+        foreach ($sites as &$site) {
+            $site['timezone_name'] = $this->getTimezoneName($site['timezone']);
+        }
```

Comment:
For consistency, this property should be added to other `getSitesWith...` methods. Would be good to put it in `getSitesFromIds()` & then a for loop after the above call to `getPatternMatchSites()`.

Reply:
Done.